tests/unit/models/ComponentAttribute.spec.js

```diff
@@ -307,7 +307,10 @@ describe('Test class: ComponentAttribute', () => {
       it('Check if respect all conditions', () => {
```

Comment:
Put test with according class (in commit)

Reply:
done